postgres/osmap.yaml

```diff
@@ -85,15 +90,30 @@ RedHat:
 
 {% else %}
 
+  {% set data_dir = '/var/lib/pgsql/data' %}
   pkg: postgresql-server
   pkg_client: postgresql
 
 {% endif %}
 
+  service: postgresql-{{ repo.version }}
+  pkgs_extra: [ postgresql-contrib, postgresql-plpython, postgresql-jdbc, postgresql-docs]
+  conf_dir: {{ data_dir }}
+  prepare_cluster:
+    command: initdb --pgdata='{{ data_dir }}'
+    test: test -f '{{ data_dir }}/PG_VERSION'
+    user: postgres
+
 Suse:
   pkg: postgresql-server
   pkg_client: postgresql
   pkg_libpq_dev: postgresql
+  pkgs_extra: [ postgresql-contrib, postgresql-plpython, postgresql-jdbc, postgresql-docs]
 
+  {% set data_dir = '/var/lib/pgsql/data' %}
```

Comment:
If you're going to set global variable, better to put it on top of the file. It would be redefined in `if` statement when necessary. But you don't need it at all.

Reply:
agreed.